fhir-persistence/src/test/java/com/ibm/fhir/persistence/search/test/AbstractSearchQuantityTest.java

```diff
@@ -170,163 +199,200 @@ public void testSearchQuantity_Quantity_NoDisplayUnit() throws Exception {
     }
-    
+
     @Test
     public void testSearchQuantity_Quantity_NoCodeOrUnit() throws Exception {
         // spec isn't clear about whether quantities with no unit should be indexed
         // but since we require the unit while searching, it doesn't really matter
         assertSearchDoesntReturnSavedResource("Quantity-noCodeOrUnit", "1||eq");
     }
-    
+
     /***
-     * FHIR Server does not yet use quantity comparator to calculate search results. *
+     * FHIR Server does not yet use quantity comparator to calculate search results.
+     * *
      *********************************************************************************/
     // Quantity search is of the form <prefix><number>|<unit_system>|<unit>.
     // We use custom units to mark the quantity comparators so we can scope our searches in the tests.
-    
+
     @Test
     public void testSearchQuantity_Quantity_LessThan() throws Exception {
         // Later versions of the spec indicate that there is an implicit precision 
         // of .5 of the next least significant digit.  We don't support that now, but 
         // lets use numbers far enough away that it won't matter.
-//        assertSearchReturnsSavedResource("Quantity-lessThan", "2||lt");
+        //        assertSearchReturnsSavedResource("Quantity-lessThan", "2||lt");
         assertSearchDoesntReturnSavedResource("Quantity-lessThan", "4||lt");
-        
+
         // With implicit ranges, 3 (+/-0.5) actually might be < 3
-//      assertSearchDoesntReturnSavedResource("Quantity-lessThan", "3||lt");
-        
-//        assertSearchReturnsSavedResource("Quantity-lessThan", "lt2||lt");      // < 3 may be < 2
-        assertSearchReturnsSavedResource("Quantity-lessThan", "gt2||lt");      // < 3 may be > 2
-        assertSearchReturnsSavedResource("Quantity-lessThan", "lt4||lt");      // < 3 may be < 4 
+        //      assertSearchDoesntReturnSavedResource("Quantity-lessThan", "3||lt");
+
+        //        assertSearchReturnsSavedResource("Quantity-lessThan", "lt2||lt");      // < 3 may be < 2
+        assertSearchReturnsSavedResource("Quantity-lessThan", "gt2||lt"); // < 3 may be > 2
+        assertSearchReturnsSavedResource("Quantity-lessThan", "lt4||lt"); // < 3 may be < 4 
         assertSearchDoesntReturnSavedResource("Quantity-lessThan", "gt4||lt"); // < 3 is not > 4
     }
-    
+
     @Test
     public void testSearchQuantity_Quantity_GreaterThan() throws Exception {
         // Later versions of the spec indicate that there is an implicit precision 
         // of .5 of the next least significant digit.  We don't support that now, but 
         // lets use numbers far enough away that it won't matter.
         assertSearchDoesntReturnSavedResource("Quantity-greaterThan", "2||gt");
-//        assertSearchReturnsSavedResource("Quantity-greaterThan", "4||gt");
-        
+        //        assertSearchReturnsSavedResource("Quantity-greaterThan", "4||gt");
```

Comment:
can this be uncommented?

Reply:
or is it commented still because we don't consider Quantity comparators?

Reply:
Right no - comparators.